jdaviz/app.py

```diff
@@ -1290,9 +1321,9 @@ def _on_new_viewer(self, msg, vid=None, name=None):
         viewer : `~glue_jupyter.bqplot.common.BqplotBaseView`
             The new viewer instance.
         """
-
         viewer = self._application_handler.new_data_viewer(
             msg.cls, data=msg.data, show=False)
+        viewer.session.parent_app = self
```

Comment:
Is this what you mean at https://github.com/glue-viz/glue-jupyter/pull/274#issuecomment-991727167 and https://github.com/glue-viz/glue-jupyter/pull/274#issuecomment-1021300876 , @astrofrog and @maartenbreddels ? Because it does not work. More to follow.